src/v/storage/types.h

```diff
@@ -31,6 +31,7 @@ namespace storage {
 using log_clock = ss::lowres_clock;
 using debug_sanitize_files = ss::bool_class<struct debug_sanitize_files_tag>;
 
+// These must be in order of increasing severity, with ok the default.
```

Comment:
This change is unrelated

Reply:
I thought I added this in the same commit as the first use of std::max() on the enum.

Reply:
This has been addressed.